### ISSUE TITLE
FR: Support for Remarkable Paper Pro

### ISSUE THREAD

**Issue by ProfesseurIssou** [koreader/koreader#12856]
Hello,

I recently purchased a Remarkable Paper Pro, and I am eager to enhance its capabilities by installing the Koreader to enjoy reading a variety of file formats, such as PDF, EPUB, CBZ, and CBR. Unfortunately, I have encountered difficulties finding a tutorial or support documentation that applies to my device, specifically the version 3.15.4.2 of the operating system.

Feature Request:

Could you please add support or provide a guide to install the Koreader on the Remarkable Paper Pro, especially for OS version 3.15.4.2? This feature would greatly improve the versatility of the device for users who wish to read a wide range of digital formats seamlessly.

Details:

    Device Information:
        Model: Remarkable Paper Pro
        OS Version: 3.15.4

    Desired Formats:
-         PDF
-         EPUB
-         CBZ
-         CBR

Request for Developers/Community:

    Provide compatibility for OS version 3.15.4.
    Share a clear tutorial or installation instructions tailored for this specific device and OS.

Thank you for considering this feature request. I look forward to any updates or guidance on this matter!

I'm available to test a possible version
Best regards

**Comment by Frenzie** [comment]
You saw https://github.com/koreader/koreader/wiki/Installation-on-ReMarkable ?

**Comment by ProfesseurIssou** [comment]
yes but toltec is not compatible with my version, and the manual installation are for remarkable 2 only

**Comment by pazos** [comment]
We won't be able to get support here until the remarkable hacking community figures out how to deal with the device.

I'll suggest to try to contact active developers that have the hardware. Maybe it's too soon for the new remarkable as a hacking platform?

**Comment by MetasharpDev** [comment]
The Paper Pro has an official developer mode. Maybe it's the way to go.

**Comment by NiLuJe** [comment]
Doesn't help us refresh the screen, AFAIK ;).

**Comment by NoaHimesaka1873** [comment]
@NiLuJe Per rM unofficial dev Discord, rM1/2 version *does* work (with FULL COLOUR!) with qtfb shim, but there's some caveats:
* Blurry text presumably due to mismatched resolution
* It always detects disk as being full
* Wi-Fi switching can put it into limbo state

**Comment by pazos** [comment]
@NoaHimesaka1873: do you have a RMPP?

It looks like somebody built an aarch64 version for the remarkable. That alone without device id it is unlikely to work. So they might have made other changes.

You can talk to discord people and point them here. Device (here!) needs a new toolchain, new properties and making sure the input translation works properly. AFAICT it can be run with https://github.com/asivery/rmpp-qtfb-shim pretty much like the RM2 runs on top of https://github.com/ddvk/remarkable2-framebuffer

**Comment by NoaHimesaka1873** [comment]
@pazos not yet, but getting one pretty soon so that's why I'm following updates for this thing.
